operator/src/index.js

```diff
@@ -7,8 +7,31 @@ const {Store} = require("./store");
 
 const port = process.env.WS_PORT;
 
-async function startServer(provider, enigmaUrl, contractAddr, scAddr, threshold, accountIndex = 0) {
-    const api = new OperatorApi(provider, enigmaUrl, contractAddr, scAddr, threshold, accountIndex);
+// This method tries to create an ethereum account from a private key provided from the environment.
+// If no private key is found, it attempts to fetch the unlocked accounts from the ethereum node,
+// and chooses the one whose index has been passed in the constructor as accountIndex.
+// If that fails too, an error is thrown.
+async function configureWeb3Account(web3) {
+    let address;
+    if (process.env.OPERATOR_ETH_PRIVATE_KEY) {
+        const account = web3.eth.accounts.privateKeyToAccount(process.env.OPERATOR_PRIVATE_KEY);
+        web3.eth.accounts.wallet.add(account);
+        address = account.address;
```

Comment:
For the reviewer:
This environment variable will be passed in to the server in the k8s cluster